worldedit-core/src/main/java/com/sk89q/worldedit/command/SelectionCommands.java

```diff
@@ -208,7 +210,8 @@ public void hpos2(Player player, LocalSession session) throws WorldEditException
     @CommandPermissions("worldedit.selection.chunk")
     public void chunk(Player player, LocalSession session,
                       @Arg(desc = "The chunk to select", def = "")
-                          BlockVector2 coordinates,
+                      @Chunk3d
+                          BlockVector3 coordinates,
```

Comment:
This looks like a breaking change to me, not sure if important.

Reply:
It's not considered API, so not dev-breaking, and due to the Chunk3d annotation, it also parses old syntax, so not user-breaking.